### PR TITLE
Implemented Quota table import/export

### PR DESCRIPTION
**Pull request by ElliotKwan**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -219,8 +219,10 @@
             <br>
             <br>
         </form>
-    	<button type="submit" class="btn btn-primary mb-2" id="makeNewQ" onClick=CreateNewQuota_DOM(document.getElementById("QuotaRows").value) >Create New Quota</button>
-
+        <button type="submit" class="btn btn-primary mb-2" id="makeNewQ" onClick=CreateNewQuota_DOM(document.getElementById("QuotaRows").value) >Create New Quota</button>
+        <button type="submit" class="btn btn-primary mb-2" id="exportQuotas" onClick=ExportQuotas() >Export Quotas</button>
```
Comment (KGSDilshan) [inline index.html]: Can you create the export button only if the import has occurred OR the sample has been flagged?

File: quotas/quotus.js [hunk 2]
```diff
@@ -40,10 +40,54 @@ function ReadQuotaTables() {
             data += qBuff.childNodes[i].childNodes[0].innerText + "\n";
         }
     }
-    data = data.split("\n");
```
Comment (KGSDilshan) [inline quotas/quotus.js]: I'd leave the logic here. Some functions are depending on it being `\n` split. Rather, in your function using it, do:
```javascript
ReadQuotaTables().join("");
```

File: index.html (hunk 1 above)
Comment (KGSDilshan) [inline index.html:225]: Can you create the export button only if the import has occurred OR the sample has been flagged?

File: quotas/quotus.js [hunk 3]
```diff
@@ -40,10 +40,54 @@ function ReadQuotaTables() {
...
-    console.log(data);
+    //data = data.split("\n");
```
Comment (KGSDilshan) [inline quotas/quotus.js]: Here don't change the function logic. Some functions are looking for the data to be split by new line. Rather in your function:
```javascript
let myData = ReadQuotaTables().join("")
```
for the same behaviour.

**Review by KGSDilshan — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (ElliotKwan) [reply index.html:225]: Can the user export quotas just by adding them in without having to first import a file or uploading a sample?

File: quotas/quotus.js (hunk 3 above)
Comment (ElliotKwan) [reply quotas/quotus.js]: OK- I wasn't aware of anything else using it because the return wasn't there, that's my bad

File: index.html (hunk 1 above)
Comment (KGSDilshan) [reply index.html:225]: Good point. Having a check to see if there is data to export is sufficient.

**Pull request merged**